Doc/howto/unicode.rst

```diff
@@ -451,6 +392,91 @@ other".  See
 list of category codes.
 
 
+Lowercasing Strings
+-------------------
+
+The string :meth:`~str.casefold` method converts a string to lowercase
```

Comment:
I'm not sure that it is correct to call str.casefold a method converting a string to lowercase. It converts a string into a case-insensitive form for using in case-insensitive comparing. AFAIK there is no other application of it. The result is not purposed for displaying to the end user. It could be an uppercase form or a sequence of integers.

Reply:
I've dropped the lowercasing section and moved the casefold() discussion into the case-insensitive comparison section.